config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main.yaml

```diff
@@ -453,3 +453,107 @@ periodics:
     testgrid-tab-name: capz-periodic-100-nodes-k8s-master
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
     description: Runs scalability tests with 100 nodes on latest cluster-api-provider-azure
+- name: periodic-cluster-api-provider-azure-windows-with-ci-artifacts
+  cluster: eks-prow-build-cluster
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  minimum_interval: 72h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-community: "true"
+  extra_refs:
```

Comment:
Does this not need cluster-api-provider-azure as an extra ref like the other tests in this file?

Reply:
Nice catch, yes, thanks!